pkg/providers/docker/reconciler/reconciler.go

```diff
@@ -76,16 +75,15 @@ func (r *Reconciler) ReconcileWorkerNodes(ctx context.Context, log logr.Logger,
 }
 
 // ReconcileWorkers applies the worker CAPI objects to the cluster.
-func (r *Reconciler) ReconcileWorkers(ctx context.Context, log logr.Logger, clusterSpec *cluster.Spec) (controller.Result, error) {
+func (r *Reconciler) ReconcileWorkers(ctx context.Context, log logr.Logger, spec *cluster.Spec) (controller.Result, error) {
```

Comment:
any reason for the clusterSPec to spec change? consistency throughout?

Reply:
yeah, it's just shorter and I we don't need to distinguish between different specs here